docs/en/developer/api.rst

```diff
@@ -0,0 +1,231 @@
+API documentation
+=================
+
+Thanks to this documentation, we'll see how to interact with the wallabag API.
+
+Requirements
+------------
+
+* wallabag freshly installed on http://localhost:8000
+* ``httpie`` installed on your computer (`see project website <https://github.com/jkbrzt/httpie>`__)
+* all the API methods are documented here http://localhost:8000/api/doc
+
+Create new API client
+---------------------
+
+In your wallabag account, you can create a new API client at this URL ``http://localhost:8000/developer/client/create``.
+
+Just give the redirect URL of your application and create your client.
+
+You have information like this:
+
+::
+
+    Client ID:
+
+    1_3o53gl30vhgk0c8ks4cocww08o84448osgo40wgw4gwkoo8skc
+
+    Client secret:
+
+    636ocbqo978ckw0gsw4gcwwocg8044sco0w8w84cws48ggogs4
+
+
+Create refresh token
+--------------------
+
+For each API call, you need a token. Let's create it via this command (replace ``client_id``, ``client_secret``, ``username`` and ``password`` values):
```

Comment:
you need --> you'll need
via this command --> with this command
values --> with their values